server/src/app/utils/crypto.utils.ts

```diff
@@ -16,8 +16,8 @@ export class CryptoUtils {
     }
   }
 
-  public comparePasswords = async (firstPw: string, secondPw: string): Promise<void> => {
-    const result = await bcrypt.compare(firstPw, secondPw);
+  public comparePasswords = async (firstPw: string, secondPw: string, pwIsToken: boolean): Promise<void> => {
+    const result = pwIsToken ? firstPw === secondPw : await bcrypt.compare(firstPw, secondPw);
```

Comment:
why did you add this? i don't get your intention.

Reply:
I needed to add this since I don't know the password of the mock users. So I couldn't login with these useless to test the authorization of the recipe endpoints. If you tell me all password I can remove this again. This method allowed me to login directly with the token instead of the password.

Reply:
the user passwords are inside the `30-socobouser.route.test.ts`file

please remove this. i think this is a big security issue if we could login with the password hash value.

Reply:
Yes I will remove this, it was only to test my routes. I didn't plan to keep this in the code.